testsuite/tests/no_std

```diff
@@ -1 +0,0 @@
-../no_std/tests/
```

Comment:
didn't you like this symlink?

I thought it would make it easier to find the `no_std` testcases.